vmdb/app/models/metric/capture/openstack_infra.rb

```diff
@@ -14,22 +16,39 @@ def self.diff_meter?(meters)
     meters.all? { |m| DIFF_METERS.include? m }
   end
 
+  # TODO(lsmola) until we have hardware.memory.util in Ceilometer, we have to compute it, but this computation
+  # should be done rather on the Ceilometer side
+  def self.memory_util_calculation(stats, _)
+    stats['hardware.memory.total'] > 0 ? 100.0 / stats['hardware.memory.total'] * stats['hardware.memory.used'] : 0
+  end
+
+  def self.counter_sum_per_s_calculation(stats, intervals)
+    total = 0.0
+    stats.keys.each do |c|
+      total += (intervals[c] > 0) ? stats[c] / intervals[c].to_f : 0
+    end
+    total
```

Comment:
In OpenStack Infra we don't have bytes. We have blocks and datagrams. We will probably change unit in the UI charts.